notebooks/concepts/imviz_dithered_gwcs.ipynb

```diff
@@ -101,7 +101,7 @@
    "outputs": [],
    "source": [
     "# Link by WCS\n",
-    "imviz.plugins['Orientation'].link_type = 'WCS'\n",
+    "imviz.plugins['Orientation'].align_by = 'WCS'\n",
     "imviz.plugins['Orientation'].wcs_use_affine = True"
```

Comment:
You might have to grep again. Might have missed some.
```suggestion
    "imviz.plugins['Orientation'].fast_approximation = True"
```

Reply:
That's the only one a grep turned up, thanks.